### DISCUSSION TITLE
Safe click with onDrag

### DISCUSSION THREAD

**Discussion by hellozyemlya** [pmndrs/use-gesture#420] [Q&A]
What is the safest way handle pointer click and prevent onDrag to be triggered? 
I have idea only with timing checking, for example, drag event not handled till N sec event length, and click not handled after N sec event length.
Maybe someone will have another ideas.

Thanks in advance.

**Comment by dbismut** [comment]
https://use-gesture.netlify.app/docs/options/#filtertaps
